src/highdicom/sr/templates.py

```diff
@@ -1215,6 +2111,40 @@ def __init__(
             value_item.ContentSequence.extend(algorithm_id)
         self.append(value_item)
 
+    @classmethod
+    def from_sequence(cls, sequence: ContentSequence) -> 'Measurement':
+        """Construct object from a sequence of content items.
+
+        Parameters
+        ----------
+        sequence: highdicom.sr.ContentSequence
+            Content Sequence containing one SR NUM Content Items
+
+        Returns
+        -------
+        highdicom.sr.Measurement
+            Content Sequence containing one SR NUM Content Items
+
+        """
+        if len(sequence) > 1:
+            raise ValueError('Measurement shall contain only one content item.')
+        item = sequence[0]
+        if not isinstance(item, NumContentItem):
+            raise TypeError('Measurement shall contain a NUM content item.')
+        measurement = cls(
+            name=item.name,
+            value=item.value,
+            unit=item.unit,
+            qualifier=item.qualifier
+        )
+        if 'ContentSequence' in item:
+            measurement.ContentSequence = ContentSequence.from_sequence(
+                item.ContentSequence
+            )
+        else:
+            measurement.ContentSequence = ContentSequence()
```

Comment:
I think having an empty content sequence like this may be invalid. 

According to [the standard](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.17.3.html#table_C.17-6):
> One or more items shall be included in this sequence.

I think it should just be omitted if the original item has no content sequence

Reply:
Good catch! Fixed via 79aa9fa1d0b1e0cd48c371affb1bc2a9798041dd